xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEResamplePi.cpp

```diff
@@ -280,7 +280,7 @@ bool CActiveAEResamplePi::Init(uint64_t dst_chan_layout, int dst_channels, int d
 
   LOGTIME(3);
 
-  if (CSettings::Get().GetBool("videoplayer.usedisplayasclock") && CSettings::Get().GetInt("videoplayer.synctype") == SYNC_RESAMPLE)
+  if (CSettings::Get().GetBool("videoplayer.usedisplayasclock"))
```

Comment:
that may be true but the dependency to the settings makes no sense. why don't you check m_src_rate and m_dst_rate